### PR TITLE
[Components with more than 1 service] LPS-189681 Refactor HttpMethodScopeLogic component 

### PR DESCRIPTION
**Pull request by Valmir-unicap**
Related: https://liferay.atlassian.net/browse/LPS-189681

Correction of: https://github.com/liferay-core-infra/liferay-portal/pull/5277

### CODE REVIEW

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Comment by Valmir-unicap** [comment]
ci:test:security

**Comment by Valmir-unicap** [comment]
ci:test:oauth2

File: modules/apps/oauth2-provider/oauth2-provider-rest/src/main/java/com/liferay/oauth2/provider/rest/internal/scope/util/HttpMethodScopeLogicUtil.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.ServiceReference;
+
+/**
+ * @author Valmir Junior
+ */
+public class HttpMethodScopeLogicUtil {
+
+	public static boolean check(
+		Function<String, Object> propertyAccessorFunction, String requestMethod,
```
Comment (rafaprax) [inline modules/apps/oauth2-provider/oauth2-provider-rest/src/main/java/com/liferay/oauth2/provider/rest/internal/scope/util/HttpMethodScopeLogicUtil.java:31]: Hey @Valmir-unicap, please sort parameters alphabetically. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by Valmir-unicap** [comment]
ci:test:sf

File: modules/apps/oauth2-provider/oauth2-provider-rest/src/main/java/com/liferay/oauth2/provider/rest/internal/scope/logic/HttpMethodScopeLogic.java [hunk 2]
```diff
@@ -49,58 +36,9 @@ public boolean check(
...
-				_log.warn(exception);
-			}
-		}
-
-		return false;
-	}
-
-	@Activate
-	protected void activate(
-		BundleContext bundleContext, Map<String, Object> properties) {
```
Comment (MrJohn1911) [inline modules/apps/oauth2-provider/oauth2-provider-rest/src/main/java/com/liferay/oauth2/provider/rest/internal/scope/logic/HttpMethodScopeLogic.java:98]: Hey @Valmir-unicap, please don't erase this method.

**Review by MrJohn1911 — CHANGES_REQUESTED** [review]

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:relevant

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Comment by Valmir-unicap** [comment]
ci:test:oauth2

**Comment by Valmir-unicap** [comment]
ci:test:security

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:relevant

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Comment by Valmir-unicap** [comment]
ci:test:oauth2

**Comment by Valmir-unicap** [comment]
ci:test:security

**Review by MrJohn1911 — APPROVED** [review]

**Comment by catarineaguiar** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
ci:test:startup

**Comment by catarineaguiar** [comment]
ci:test:security

**Comment by catarineaguiar** [comment]
no regressions found

**Comment by shuyangzhou** [comment]
See https://github.com/liferay-core-infra/liferay-portal/pull/5421

**Pull request closed without merging**